app/controllers/wikis_controller.rb

```diff
@@ -1,20 +1,38 @@
 class WikisController < ApplicationController
-  skip_before_action :authenticate_user!, except: []
+  #skip_before_action :authenticate_user!, only: [:index, :show]
+  before_action :authenticate_user!, except: [:index, :show]
+  after_action :verify_policy_scoped, only: :index
+  after_action :verify_authorized, only: [:create, :new]
```

Comment:
I'm going to merge this, but you probably don't need these two lines. When you call `policy_scope` and `authorize` in the controller methods they do the same as these and are preferable.